### ISSUE TITLE
Support v5

### ISSUE THREAD

**Issue by rienheuver** [florian-lefebvre/astro-i18n-test#42]
Astro v5 is now the default version but installing this package doesn't work with it yet. I'm now trying to install it nonetheless with `npm i (...) --force`, I'll update here if that works or not (if I won't forget).

**Comment by rienheuver** [comment]
I can't get it to work. Not sure if this has to do with v5, but when following the installation and usage instructions I quickly run into `ENOENT: no such file or directory, scandir '/<my-repo-path>/src/routes'` and there's nothing I can find in the documentation as to why that would happen. There's something about `/pages` and `/routes` in the documentation here: https://astro-i18n.netlify.app/getting-started/usage/
But what the distinction is between those two folders and when to use which? No idea. My site uses `/pages`, so no idea what the routes directory is for.

Also, when I rename my pages directory to routes (to see if that fixes it), I get an import error in one of my page files. It relies on a script that it imports from a few directories up in the tree, but because the pages is getting build to something prefixed, that path is broken.

**Comment by florian-lefebvre** [comment]
Hi thanks for opening an issue! First of all, let me warn you I'm not actively maintaining this integration by lack of time.

I'll try to make it work with 5.0, I don't think it should be affected by many (if any) breaking changes. As for the error you get, I think it could be unrelated to v5. Could you provide a minimal reproduction?

Thank you!

**Comment by rienheuver** [comment]
Sure thing: https://github.com/rienheuver/i18n-v5-broken
Reproduce:
```
git clone https://github.com/rienheuver/i18n-v5-broken
npm i
npm run dev
```

To be a bit more clear, this repo was simply made with:
```
npm create astro@latest
cd my-project/
npm install @astrolicious/i18n i18next --force
```
and then adding i18n to `astro.config.mjs` and creating a `locales` dir with a `common.json` per locale

**Comment by rienheuver** [comment]
I also tried to reproduce the importing bug I mentioned, but got an error: `ENOENT: no such file or directory, open '/my-project/src/env.d.ts'`

I manually created an empty file `env.d.ts` in that location which resolved the bug. I was then able to reproduce the import bug. It's on the same repo under the `import-bug` branch: https://github.com/rienheuver/i18n-v5-broken/tree/import-bug

Let me know if there's more I can do!
